Source/Evolve/WarpXEvolve.cpp

```diff
@@ -370,6 +370,7 @@ WarpX::Evolve (int numsteps)
 
         exit_loop_due_to_interrupt_signal = SignalHandling::TestAndResetActionRequestFlag(SignalHandling::SIGNAL_REQUESTS_BREAK);
         if (cur_time >= stop_time - 1.e-3*dt[0] || exit_loop_due_to_interrupt_signal) {
+            ExecutePythonCallback("onbreaksignal");
```

Comment:
Note that this will call `onbreaksignal` if the stop time is reached. Is this intended? Should this be
```
if (exit_loop_due_to_interrupt_signal) ExecutePythonCallback("onbreaksignal");
```

Reply:
Yes, it should be. Thanks for catching that!

Reply:
In my opinion it would be better to move this callback execution down a couple lines (to 385) so that the callbacks execute after the diagnostic output. That would also make it more consistent with the proposed documentation saying that the `onbreaksignal` callback executes lasts before the code exits.